src/main/java/org/squiddev/cobalt/lib/MathLib.java

```diff
@@ -53,15 +53,15 @@ public LuaValue add(LuaState state, LuaTable env) {
 		t.rawset("pi", ValueFactory.valueOf(Math.PI));
 		t.rawset("huge", LuaDouble.POSINF);
 		LibFunction.bind(t, MathLib1::new, new String[]{
-			"abs", "ceil", "cos", "deg",
-			"exp", "floor", "rad", "sin",
-			"sqrt", "tan",
-			"acos", "asin", "atan", "cosh",
-			"exp", "log", "log10", "sinh",
-			"tanh"
+				"abs", "ceil", "cos", "deg",
```

Comment:
Sorry to be a grumpy grump, but can you undo the indentation change here? For whatever reason, IntelliJ defaults to a "continuation indent" or 8 rather than 4.

Reply:
Of course, my bad